docs/Usage.md

```diff
@@ -121,11 +121,11 @@ The GlosSI overlay can also be navigated with controller-inputs
 
     Rename the shortcut in Steam to the Steam-AppID of the game you want to access to community-configs.  
     Community configs may only be available **before** you launch the shortcut.  
-    AppIDs can be retrieved from [SteamDB](https://steamdb.info/apps/)
+    AppIDs can be retrieved from [Steambase](https://steambase.io/)
```

Comment:
@AdonisCodes - I don't think the existing url should be removed. ie: **SteamDB or Steambase**. Also, can you update the url to be `http://steambase.io/apps/`